<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdditionalDatasetOptions</class>
 <widget class="QDialog" name="AdditionalDatasetOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compute additional dataset</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="alphaEditor">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>158</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="alphaLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Alpha</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="gammaEditor">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>158</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <double>2.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="gammaLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gamma</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="betaEditor">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>158</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <double>0.500000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="betaLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Beta</string>
   </property>
  </widget>
  <widget class="QLabel" name="sigmaLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sigma</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sigmaEditor">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AdditionalDatasetOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AdditionalDatasetOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
